app/pubspec.lock

```diff
@@ -78,6 +78,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.5"
+  csslib:
```

Comment:
CSS and html? I did not need this when attempting email with PDF.

- [ ] Remove this if we don't need them